lib/interface/cli/commands/trigger/type/get.cmd.js

```diff
@@ -0,0 +1,50 @@
+const debug = require('debug')('codefresh:cli:get:trigger-types');
+const Command = require('../../../Command');
+const _ = require('lodash');
+const { trigger } = require('../../../../../logic').api;
+const { specifyOutputForSingle, specifyOutputForArray } = require('../../../helpers/get');
+const getRoot = require('../../root/get.cmd');
+
+const command = new Command({
+    command: 'trigger-types [type] [kind]',
+    aliases: ['tt'],
+    parent: getRoot,
+    cliDocs: {
+        description: 'Get list of trigger types or individual trigger installed in Codefresh environment',
+    },
+    webDocs: {
+        category: 'Triggers',
+        title: 'Get available trigger type/s',
+    },
+    builder: (yargs) => {
+        return yargs
+            .positional('type', {
+                describe: '[optional] trigger type (e.g. registry, cron)',
```

Comment:
please add 'choices' to the object and put all possible values.
please remove [optional] sign
```
.positional('typpe', {
  describe: 'trigger type',
  choices: ['registry', 'cron']
}
```

Reply:
hard. it should be dynamic. It will be possible to add more types and optionally kinds of trigger-types, so it's hard to model them as static "options" list. Maybe use string?